src/components/user/userDAL.js

```diff
@@ -32,6 +32,18 @@ class UserDAL {
             throw e;
         }
     }
+
+    async getMatchesByID(id) {
+        user = await getById(id)
+        if(user.type === 'jobseeker'){
+          employees = this.userModel.findAll({where: type: 'employee'}).then(user => {return user;})
+          employee_interests = employees.map(e => {"id": e.id, "interests": e.interests})
+          // Regn ut jaccard/cosine av interessene til bruker mot hver employee
+          // må antageligvis ha totalt antall mulige interesser for å beregne dette
+          // returner top 30 employees med høyest likhet
```

Comment:
https://en.wikipedia.org/wiki/Cosine_similarity
Vi mapper interesser til 1, og fravær av interesser til 0.

Gitt to brukere med følgende interesser: 
- `{'hund', 'katt', båt}`
-`{'hund', 'båt', 'hus', 'bil'}`
Mapper om til "vektorer":
- Bruker 1:
  - `{'hund': 1, 'katt': 1, 'båt': 1, 'hus' :0, 'bil': 0}`
- Bruker 2:
  - `{'hund': 1, 'katt': 0, 'båt': 1, 'hus' :1, 'bil': 1}`

Disse rekkene med 1 og 0 vil så sammenlignes gitt formelen for cosine-likhet, og gi et tall som angir likheten mellom brukerne. Det kan dog hende at dette er for dårlig på brukere med få interesser, kan vurdere andre likhetsmål (jaccard, td-idf)